static/js/popovers.js

```diff
@@ -700,7 +700,7 @@ exports.show_sender_info = function () {
 // side effect of closing popovers, which we don't want.  So we
 // suppress the first hide from scrolling after a resize using this
 // variable.
-let suppress_scroll_hide = false;
+let suppress_scroll_hide = true;
```

Comment:
I am not sure how I feel about changing this logic, which is specifically related to mobile web.  Are we sure this change is necessary after we've fixed the initialization order?

Reply:
Before changing the initialization order, the hide_all function was called multiple times. After changing the initialization order, it works correctly if we initialize zulip on an empty stream( i.e a stream with no messages in it), but causes problems if it is called on a stream with messages. This is because, the messages cause the view to scroll and change while being they are being displayed. Changing the initialization order does not solve the message loading issue. This answers your question. https://chat.zulip.org/#narrow/stream/9-issues/topic/User.20info.20popover.20does.20not.20hide.20.20on.20opening.20menu/near/857243

So setting this to true is necessary to prevent the popovers.hide_all being called the first time. Other modifications to the ('app').on('scroll') are required for the same reason. prevent the scroll function from being called multiple times.

Reply:
One thing we could do is remove the gear_initialize from the ui_init and initialize the gear icon after all the messages have been reloaded? Will that approach work?